doubleml/double_ml.py

```diff
@@ -409,22 +415,14 @@ def __psi(self):
         return self._psi[:, self._i_rep, self._i_treat]
 
     @property
-    def __psi_a(self):
-        return self._psi_a[:, self._i_rep, self._i_treat]
-
-    @property
-    def __psi_b(self):
-        return self._psi_b[:, self._i_rep, self._i_treat]
-
-    @property
-    def __all_coef(self):
-        return self._all_coef[self._i_treat, self._i_rep]
+    def __psi_deriv(self):
+        return self._psi_deriv[:, self._i_rep, self._i_treat]
 
     @property
     def __all_se(self):
         return self._all_se[self._i_treat, self._i_rep]
 
-    def fit(self, n_jobs_cv=None, keep_scores=True, store_predictions=False):
+    def fit(self, n_jobs_cv=None, store_predictions=False):
```

Comment:
I removed the `keep_scores` option in this PR. My feeling was that it is anyways rarely used and just made things more complicate.